kernel/packages/unity-interface/UnityScene.ts

```diff
@@ -24,6 +25,15 @@ export class UnityScene<T> implements ParcelSceneAPI {
 
   constructor(public data: EnvironmentData<T>) {
     this.logger = DEBUG_SCENE_LOG === true ? createLogger(getParcelSceneID(this) + ': ') : createDummyLogger()
+
+    const startLoadingTime = performance.now()
+
+    this.eventDispatcher.once('sceneStart', () => {
```

Comment:
We should divide this tracking event in two, `scene_start` and `scene_end`. Also let's use snake_case to be more consistent. cc: @sicarul

Reply:
Line 32 has it `scene_start_event` the `sceneStart` is the event sent by unity to the SDK